### PR TITLE
feat(server): data persistence

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/config/initializers/ckb/utils.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      while current_from <= to
+        current_to = [current_from + 100, to].min
+        cells = CkbSync::Api.get_cells_by_lock_hash(lock_hash, current_from, current_to)
+        results.concat(cells)
+        current_from = current_to + 1
+      end
+      results
+    end
+
+    #TODO 可以改成通过本地的 cell 来计算。
```
Comment (duanyytop) [inline server/config/initializers/ckb/utils.rb]: Annotation should be english as long as possible.

File: server/.env [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # database config
 DB_USERNAME="postgres"
 DB_PASSWORD="postgres"
+CKB_NODE_URL="http://localhost:3333"
```
Comment (duanyytop) [inline server/.env]: CKB node url port is 8114.

Comment (duanyytop) [inline server/.env]: CKB node url port is 8114.

Comment (shaojunda) [reply server/.env]: this is just an example.

File: server/config/initializers/ckb/utils.rb (hunk 1 above)
Comment (shaojunda) [reply server/config/initializers/ckb/utils.rb]: ok

File: server/.env (hunk 2 above)
Comment (duanyytop) [reply server/.env]: Preciseness will be better.

File: server/app/models/block.rb [hunk 3]
```diff
@@ -1,6 +1,249 @@
...
+  def difficulty=(difficulty)
+    super([difficulty[2..-1]].pack("H*"))
+  end
+
+  def block_hash
+    "0x#{super.unpack("H*").first}"
+  end
+
+  def block_hash=(block_hash)
+    super([block_hash[2..-1]].pack("H*"))
```
Comment (classicalliu) [inline server/app/models/block.rb]: Prefer `block_hash.delete_prefix("0x")` to instead `block_hash[2..-1]`

File: server/config/initializers/ckb/utils.rb (hunk 1 above)
Comment (classicalliu) [reply server/config/initializers/ckb/utils.rb]: Update this.

**Pull request merged**